crates/optimism/cli/src/commands/import_receipts.rs

```diff
@@ -221,23 +199,54 @@ where
             // this ensures the execution outcome and static file producer start at block 1
             first_block = 1;
         }
+        highest_block_receipts = first_block + receipts.len() as u64 - 1;
+
+        // RLP file may have too many blocks. We ignore the excess, but warn the user.
```

Comment:
when would this happen?
if the file has more blocks than the bedrock transition block? or duplicated blocks?

> There seems to exist an extra block 105235064